### DISCUSSION TITLE
Release a v1.9.3 tag

### DISCUSSION THREAD

**Discussion by Simon-Lopez** [wjakob/nanobind#520] [Q&A]
Hello all,

The fix of #475 (afe2b83cab5ffe00feba3b0782eabf1e5ea8de13) is really usefull to compile with C++23 standard on.
Would it be possible to have a v1.9.3 tag on the main branch to have a wheel released on pipy, please?
This would make specifications of dependencies much easier!

Thanks a lot !

**Comment by Seairth** [comment]
Based on a number of the other changes involved, it's a minor number increase, at least.  Regardless, I too am waiting on a release to get access to the new `sig` struct and `stubgen` features.

**Comment by wjakob** [comment]
v2.0.0 is out now

**Reply by Simon-Lopez** [reply]
Thanks !
